demo/run.sh

```diff
@@ -74,14 +74,16 @@ function runAnsible() {
   echo '-----'
 }
 
-function RetrieveSecretInTarget() {
+function retrieveSecretInTarget() {
```

Comment:
The function name changed to lower case, but the call to it is still with upper case